src/utils/set-about-panel.ts

```diff
@@ -0,0 +1,26 @@
+import { AboutPanelOptionsOptions, app } from 'electron';
```

Comment:
It's really called `AboutPanelOptionsOptions`? :laughing:

Reply:
That's um, questionable on my part 😆 The TS method takes `${methodName}${paramName}` as the default.  In this case the method is `setAboutPanelOptions` and the param is `options` 🤔 Maybe it should prevent `OptionsOptions` 🤔